<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignModePanel</class>
 <widget class="QWidget" name="SignModePanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>765</width>
    <height>395</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Работа с электронной цифровой подписью</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Создать</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="3" column="0">
        <widget class="QLineEdit" name="leSignPath"/>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Файл для подписи:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Сохранить публичный ключ как:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Сохранить приватный ключ как:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="btnBrowseSign">
         <property name="text">
          <string>Сохранить как</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QRadioButton" name="rbGenNewKey">
         <property name="text">
          <string>Сгенерировать новый приватный ключ</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="8" column="0" colspan="2">
        <widget class="Line" name="line">
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="12" column="0" colspan="2">
        <widget class="Line" name="line_2">
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QRadioButton" name="rbUseKey">
         <property name="text">
          <string>Использовать собственный приватный ключ</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QLineEdit" name="leKeyPath">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="14" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="btnBrowseFile">
         <property name="text">
          <string>Выбрать файл</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLineEdit" name="leFilePath"/>
       </item>
       <item row="5" column="1">
        <widget class="QPushButton" name="btnBrowsePubKeyPath">
         <property name="text">
          <string>Сохранить как</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Сохранить подпись как:</string>
         </property>
        </widget>
       </item>
       <item row="11" column="1">
        <widget class="QPushButton" name="btnBrowseKey">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Выбрать ключ</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLineEdit" name="lePubKeyPath"/>
       </item>
       <item row="13" column="0" colspan="2">
        <widget class="QPushButton" name="btnCreateSign">
         <property name="text">
          <string>Создать подпись</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLineEdit" name="lePrKeyPath">
         <property name="enabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QPushButton" name="btnBrowsePrKeyPath">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Сохранить как</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Проверить</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="3" column="0">
        <widget class="QLineEdit" name="leSignPathV"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Файл:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="btnBrowseSignV">
         <property name="text">
          <string>Выбрать подпись</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLineEdit" name="leKeyPathV"/>
       </item>
       <item row="5" column="1">
        <widget class="QPushButton" name="btnBrowseKeyV">
         <property name="text">
          <string>Выбрать ключ</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Подпись:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLineEdit" name="leFilePathV"/>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="btnBrowseFileV">
         <property name="text">
          <string>Выбрать файл</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Публичный ключ</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Результат проверки:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leVerifyResult">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="8" column="0" colspan="2">
        <widget class="QPushButton" name="btnVerifySign">
         <property name="text">
          <string>Проверить подпись</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="2">
        <widget class="Line" name="line_3">
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
